protostar/testing/environments/fuzz_test_execution_environment.py

```diff
@@ -127,7 +144,22 @@ def build_and_run_test(
         given_strategies: Dict[str, SearchStrategy],
     ):
         try:
-
+            functions = [
+                item
+                for item in self.state.contract.abi
+                if item["name"] == function_name
+            ]
+            assert len(functions) == 1
+            function = functions[0]
+            function_inputs_names = [input["name"] for input in function["inputs"]]
+            examples = []
+            for example_item in self.state.config.fuzz_examples:
+                if isinstance(example_item, dict):
+                    examples.append(example_item)
+                    continue
+                examples.append(dict(zip(function_inputs_names, list(example_item))))
```

Comment:
[Style] This could be a method.

Reply:
[extract code to produce_examples](https://github.com/software-mansion/protostar/pull/947/commits/cc5625620677e0ba6e22ab0be2c1e349adb1f700)